### DISCUSSION TITLE
How to bypass autoplay policy

### DISCUSSION THREAD

**Discussion by Bialson** [tauri-apps/tauri#6833] [Q&A]
Hello. My target is to include video frame in my Tauri app (React + TS) and set it to play automaticly when video ends buffering. My app is going to work on Raspberry Pi. It is possible to bypass autoplay policy in WebView on Linux? I've read that is possible on Windows but I don't know how to achieve this on Linux.

**Comment by FabianLars** [comment]
Unless there's a way to fake user interaction, this is currently not possible. The "real" setting must be changed in Wry. Maybe we can consider backporting this PR https://github.com/tauri-apps/wry/pull/913 which right now is v2 only.

**Comment by Bialson** [comment]
Are there another tutorials or guides for WRY except GitHub readme? It is possible to work with React and WRY?

**Reply by FabianLars** [reply]
> Are there another tutorials or guides for WRY except GitHub readme?

Nope, not that i know of.

> It is possible to work with React and WRY?

Yes, but it's much more work than using Tauri (unless you only distribute to devices owned or controlled by you in which case you could potentially skip the bundling step (like creating a `.deb` installer etc)

**Comment by Leoby19** [comment]
you can whatch here!

https://github.com/Leoby19/SoundJS-Notifier-NoClick

An important feature is that it allows triggering a notification sound without the need for a physical click on the page, bypassing autoplay restrictions on all browsers.

**Reply by lucaslealdev** [reply]
Hi my friend! Just to clarify, this "bypass" the need for the click by asking user to click. This is not what the OP was asking for.
